airbyte-integrations/connectors/source-slack/source_slack/schemas/channels.json

```diff
@@ -17,6 +17,9 @@
     "is_im": {
       "type": "boolean"
     },
+    "context_team_id": {
```

Comment:
should we update the expected records for the streams where the schema is updated?

Reply:
updated expectred records